i18n/request.ts

```diff
@@ -12,11 +12,13 @@ export default getRequestConfig(async ({ requestLocale }) => {
   }
   // Load multiple translation files
   const mainMessages = (await import(`@/messages/${locale}/en-US.json`)).default;
+  const templateMessages = (await import(`@/messages/${locale}/templates.json`)).default;
```

Comment:
added a new `.json` file for template page translations. It seems we have to merge all of these templates into one for `next-intl`. Couldn't figure out another way to do it.